internal/auth/rhsso_authenticator.go

```diff
@@ -55,7 +55,7 @@ func (rh *RHSSOAuthenticator) parseToken(userToken *jwt.Token) (User, error) {
 	}
 
 	return User{
-		Username:     claims["username"].(string),
+		Username:     claims["preffered_username"].(string),
 		Organization: claims["org_id"].(string),
```

Comment:
The organization looks as follows:
```
"organization":{"account_number":"1234679","id":"1234679"}
```
Can we fix that as well?

Reply:
ok

Reply:
do we need `account_number`? I would say we only need the id

Reply:
https://github.com/openshift/assisted-service/blob/be4e1e0545bc786461b6dd8ce39fed6bf3166f16/pkg/auth/rhsso_authenticator.go#L134